app/assets/stylesheets/header.scss

```diff
@@ -5,10 +5,18 @@ header > .dark-header > nav {
   .navbar-brand {
     font-weight: bold;
     font-size: $font-size-h3;
+    &, & * { color: $text-grey; }
```

Comment:
And why did you use the navbar-brand class for the entypo icons? `navbar-brand` is used for the brand name in the header.